Buildings/package.mo

```diff
@@ -445,6 +445,13 @@ have been <b style=\"color:blue\">improved</b> in a
                        This is for
                        <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/1502\">issue 1502</a>.
     </td>
+    <tr><td valign=\"top\">Buildings.Examples.VAVReheat.BaseClasses.VAVBranch
```

Comment:
- [ ] This is non-backward compatible, to be moved to `<!-- Non-backward compatible changes to existing components -->`.
- [ ] Also a conversion script command must be added for Dymola users in `Buildings/Resources/Scripts/Dymola/ConvertBuildings_from_7_to_8.0.0.mos`.
- [ ] A note for `Buildings.Examples.VAVReheat.BaseClasses.PartialOpenLoop` is perhaps also useful.

Reply:
Done.